frontend/src/locale/index.ts

```diff
@@ -833,9 +847,25 @@ export const messages = {
     "keynote.englishLanguage": "Inglese",
 
     "ranking.heading": "Risultati della votazione comunitaria",
-    "ranking.introduction": "Ecco qui i risultati della votazione comunitaria!",
+    "ranking.introduction": `Ecco qui i risultati della votazione comunitaria!{br}
+      Grazie a tutte persone che hanno votato e hanno cosi' contribuito a preparare il programma
+      per la prossima edizione di PyCon italia! C'erano ben {proposalNumber} proposte quest'anno da
+      scegliere inviate da ben {speakersNumber} fantastici speakers!
+`,
+    "ranking.topicComment":
+      "Abbiamo ricevuto {value} proposte per la track {name} quest'anno!",
     "ranking.seoTitle": "Ranking",
     "ranking.details": "Dettagli",
+    "ranking.stats.heading": "Alcune statistiche",
+    "ranking.stats.submissions": "{value} proposte in totale",
+    "ranking.stats.speakers": "{value} persone si sono proposte come speaker",
+    "ranking.stats.gender.women": "{value} proposte da speaker donne",
+    "ranking.stats.gender.men": "{value} proposte da speaker uomini",
```

Comment:
don't we also have other / prefer not to say? we should probably show those as well as "from unspecified genders" or some similar copy

Reply:
Something like:
 "{value} proposte da speaker di gender non specificato:
 "{value} proposals from unspecified gender speakers"
 ?
 
 And do we want to sum `other` and `prefer not to say` ?